module2/homework/transformContainers/transformContainer.cpp

```diff
@@ -0,0 +1,24 @@
+#include "transformContainer.hpp"
+
+#include <algorithm>
+
+std::map<int, std::string> removeDuplicateAndTranformToMap(const std::list<std::string>& list,
+                                                           const std::deque<int>& deq) {
+    std::list<std::string> uniqueList{list.begin(), list.end()};
```

Comment:
sort shouldnt be before unique? Otherwise you will not find duplicates, because they might be non consecutive.

Reply:
Sorry I was screw this up everything is all right ;/